docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -565,6 +565,21 @@ data class Person @BsonCreator constructor (
 ): PanacheMongoEntity()
 ----
 
+The last option is to the use the link:https://kotlinlang.org/docs/reference/compiler-plugins.html#no-arg-compiler-plugin[no-arg] compiler plugin.
+This plugin needs to be configured with a lit of annotations, then it will generate a parameterless constructor for each class annotated with them.
```

Comment:
```suggestion
This plugin needs to be configured with a list of annotations, then it will generate a parameterless constructor for each class annotated with them.
```